integrations/api_team_test.go

```diff
@@ -107,3 +107,33 @@ func checkTeamBean(t *testing.T, id int64, name, description string, permission
 	assert.NoError(t, team.GetUnits(), "GetUnits")
 	checkTeamResponse(t, convert.ToTeam(team), name, description, permission, units)
 }
+
+type TeamSearchResults struct {
+	OK   bool        `json:"ok"`
+	Data []*api.Team `json:"data"`
+}
+
+func TestAPITeamSearch(t *testing.T) {
+	prepareTestEnv(t)
+
+	teamUser := models.AssertExistsAndLoadBean(t, &models.TeamUser{}).(*models.TeamUser)
```

Comment:
Which team user are you loading here? I think you should specify one.

Reply:
It will load the first row (fixture) of team_user (I copied previous code), but I agree it makes sense to always write out for clarity. Anyhow I didn't even use teamUser except for getting user id so I removed it.

Reply:
It's not so much for clarity as for being future-proof.